roles/node/tasks/main.yml

```diff
@@ -1,25 +1,5 @@
-- name: Download node.js installation script
-  get_url:
-    url: 'https://deb.nodesource.com/setup_8.x'
-    dest: /home/{{ ansible_ssh_user }}/nodesource_setup.sh
-
-- name: make node script exectuable
-  file: dest=/home/{{ ansible_ssh_user}}/nodesource_setup.sh mode=a+x
-
-- name: Run the nodesource setup script
-  become: true
-  command: sh /home/{{ ansible_ssh_user }}/nodesource_setup.sh
-
-- name: remove exisiting package
-  become: true
-  apt:
-    name: nodejs
-    state: absent
-    purge: true
-
 - name: update and install nodejs
   become: true
   apt:
-    name: nodejs
+    name: node-yarn
```

Comment:
We should avoid loading from packages unless we can lock the version.  If we can't specify the versions, we can't guarantee that rerunning the build playbooks gives us the same results each time - i.e. the package versions can update in ways that aren't compatible with other version dependency assumptions we're making.